docs/customization/variables.mdx

```diff
@@ -156,7 +156,7 @@ In the following example, the primary color is set to red and the text color is
           <ClerkProvider
             appearance={{
               variables: {
-                colorPrimary: 'red',
+                colorPrimary: 'purple',
                 colorText: 'white',
```

Comment:
I think the biggest offender here was the `colorText` usage, which hides all of the text in the card. 
![Screenshot 2024-09-13 at 2 47 24 PM](https://github.com/user-attachments/assets/89803415-1716-483d-9669-f3eb13bb94ec)